### ISSUE TITLE
Lokinet for private transport

### ISSUE THREAD

**Issue by danrobi11** [ipfs/notes#431]
Hello, Plz consider supporting Lokinet for private network transport  

- ' Your real IP address isn't revealed to Lokinet addresses. Lokinet can carry any IP based protocol. Lokinet is like a VPN, it accepts all data types, this makes it easy to integrate with your application. All data that travels through Lokinet is encrypted and bounced through multiple nodes hiding the true origin ' https://lokinet.org/

Thanks!

**Comment by didnt1able** [comment]
I agree, this would be very interesting. But consider the size of lokinet vs the size of TOR. And tor is already in the works i believe

**Comment by danrobi11** [comment]
>  tor is already in the works i believe

Yep, [Galacteek now support tor](https://github.com/pinnaculum/galacteek/releases/tag/v0.4.41)
